reana_server/utils.py

```diff
@@ -509,23 +509,30 @@ def _get_gitlab_hook_id(project_id, gitlab_token):
     :param project_id: Project id on GitLab.
     :param gitlab_token: GitLab token.
     """
-    reana_hook_id = None
+    # FIXME: handle pagination of results
```

Comment:
Do we need the FIXME here?

Reply:
This is fixed by #684 , but I have removed it anyway